ci/Dockerfile.pdk

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.2.2
+FROM ruby:3.3-rc
```

Comment:
What is the rc version?

Reply:
ruby:3.3-rc is one of the most up-to-date images recently released ––

after re-reviewing the most recent scans in Snyk, our current ruby:3.2.2 reveals 132 (all low) vulnerabilities. ruby:3.3-rc drastically reduces this to 31 (all low) vulnerabilities